centreon/www/front_src/src/Dashboards/SingleInstancePage/Dashboard/Widgets/centreon-widget-statusgrid/src/StatusGridStandard/Tooltip/useBATooltipContent.ts

```diff
@@ -58,7 +63,9 @@ const useBATooltipContent = (
 
   const ProblematicKPIsCount = indicatorsWithProblems?.length || 0;
 
-  const health = Math.floor(((total - ProblematicKPIsCount) * 100) / total);
+  const health = equals(calculationMethod, CalculationMethodType.Impact) 
+    ? businessActivity?.currentLevel || 0
+    : Math.floor(((total - ProblematicKPIsCount) * 100) / total);
```

Comment:
I suppose that the health is only used for impact CM

Reply:
Yes, the API will return the current level value only for BAs of type Impact

Reply:
Tru. I've corrected it